rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/RxAudioPlayer.java

```diff
@@ -131,6 +133,30 @@ public void call(final SingleSubscriber<? super Boolean> singleSubscriber) {
                     }
                 }
             });
+        } else if (config.mType == PlayConfig.TYPE_URL && config.url != null &&
```

Comment:
Thanks for your PR!

Maybe we can turn this if-else-if into switch, and do some common check before it.

And you can check url with `TextUtil.isEmpty`.